test/cookbooks/lxctests/recipes/simple.rb

```diff
@@ -1,5 +1,11 @@
 require 'chef/provisioning'
+require 'chef/provisioning/lxc_driver'
+::Chef::Config.from_file '/home/ranjib/workspace/foss/chef-provisioning-lxc/.chef/knife.rb'
 with_driver 'lxc'
 # default ubuntu template will install 14.04, where chef is not well tested, lets use 12.04
-with_machine_options :template => 'ubuntu', :template_options=>['-r','precise']
-machine 'simple'
+machine 'simple' do
+  machine_options(
+    template: 'download',
```

Comment:
Just checking: is this the default?  If I run `machine 'simple' do; end` will a machine be created, or will it fail?

It would be useful to have a default.  Again, not a prereq but it's a strong recommendation so that people can quickly switch to a new driver, get something to happen, and build up from that point.